src/class-wpml-beaver-builder-translatable-nodes.php

```diff
@@ -98,15 +127,35 @@ public function get_string_name( $node_id, $field, $settings ) {
 	}
 
 	/**
-	 * @param array $node_data
-	 * @param stdClass $settings
+	 * Get wrap tag for string.
+	 * Used for SEO, can contain (h1...h6, etc.)
+	 *
+	 * @param stdClass $settings Field settings.
+	 *
+	 * @return string
+	 */
+	private function get_wrap_tag( $settings ) {
+		if ( isset( $settings->type ) && 'heading' === $settings->type ) {
+			if ( isset( $settings->tag ) ) {
```

Comment:
You can merge these two conditions into one

Reply:
@andreasciamanna done.